assignment/tool/src/webapp/vm/assignment/chef_assignments_list_assignments.vm

```diff
@@ -422,6 +436,15 @@
 									#end
 								#end
 							</td>
+							#if ($createdBy_enabled)
+								<td headers="createdBy">
+									#if ($assignment.Modifier)
+										${userDirectoryService.getUser($assignment.Modifier).getDisplayName()}
```

Comment:
instead of calling the service directly here, I think its more ideal to call to inject the modifier at the same location as createdBy_enabled is added in the action.